index.html

```diff
@@ -465,14 +479,27 @@
           Push service use
         </h3>
         <p>
-          A <code>PushSubscription</code> contains all the information needed to send a <a>push
-          message</a>. A <a>push message</a> can be delivered using the <a>web push protocol</a>.
-        </p>
-        <p>
-          The <code><a>endpoint</a></code> of a <code>PushSubscription</code> is a URI that allows
-          an <a>application server</a> to request delivery of a <a>push message</a> to a <a>push
-          subscription</a>.
+          The fields included in the <code>PushSubscription</code> is all the information needed
+          for an <a>application server</a> to send a <a>push message</a>. Push services that are
+          compatible with the Push API will be expected to provide an <a>endpoint</a> that conforms
+          to the <a>web push protocol</a>. These parameters and attributes include:
```

Comment:
"Push services that...web push protocol" feels like a `<p class=note>` more than normative text. It doesn't actually say anything that the reader can do anything with.